temporaldata/utils.py

```
@@ -27,3 +27,17 @@ def _size_repr(key: Any, value: Any, indent: int = 0) -> str:
         out = str(value)
     key = str(key).replace("'", "")
     return f"{pad}{key}={out}"
+
+
+def _validate_select_by_mask_input(mask, length):
+    if not isinstance(mask, np.ndarray):
+        raise ValueError(f"mask must be a numpy array (bool, 1D)")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)